File: php/facadePattern/Baggage.php
<?php  namespace php\facadePattern;
 /*
|========================================================================
|------------------------------------------------------------------------
| The values for the properties would usually be obtained from a database
| Values have been explicitly added for example purposes.
|------------------------------------------------------------------------
|========================================================================
*/
class Baggage {
	
	/**
	 * Total weight of baggage
	 * 
	 * @var string $totalWeight
	 */
	  private $totalWeight = '5KG';
	  
	/**
	 * Default baggage ammount
	 * 
	 * @var string $ammount
	 */
	  private $amount = '3';
	  
	/**
	 * Total weight of baggage
	 * 
	 * @var string $accountedFor
	 */
	  private $isBaggageAccountedFor = 'true';
	  
	/**
	 * Baggage ID
	 * 
	 * @var string $id
	 */
	  private $id;
	
	/**
	 * Obtain ID when object is created
	 * 
	 * @param string $passID
	 * @return void
	 */
  	public function __construct($passID) {
		$this->id = $passID;
  	}  
       /**
	* Get total weight of baggage
	* 
	* @return string
	*/ 
  	public function getTotalWeight() {
  		
  		return $this->totalWeight;
  	}

       /**
	* Get amount baggage ammout
	* 
	* @return string
	*/ 
	public function getAmount() {
		
  		return $this->amount;
  	}

       /**
	* Query all bags are accounted for
	* 
	* @return string
	*/ 
	public function getAllAccountedFor() {
		
  		return $this->isBaggageAccountedFor;
  	}

}

File: java/facadePattern/Baggage.java
package facadePattern;
/*
|========================================================================
|------------------------------------------------------------------------
| The values for the instance variables would 
|  usually be obtained from a database
|  However values have been added for the benfit of this example
|------------------------------------------------------------------------
|========================================================================
*/
public class Baggage {
	private String id;
	private String amount = "4";
	private String totalWeight = "3KG";
	private String allAccountedFor = "true";

	public Baggage(String id) {
		this.id = id;
	}

	public String getAmount() {
		return this.amount;
	}

	public String getTotalWeight() {
		return this.totalWeight;
	}

	public String getAllAccountedFor() {
		return this.allAccountedFor;
	}
}
/*
|========================================================================
|------------------------------------------------------------------------
|	Some methods have not been added for convinience
|	as it is only an example to display the basics.
| 
|------------------------------------------------------------------------
|========================================================================
*/
